cmd/bulk_query_gen/main.go

```diff
@@ -319,64 +319,6 @@ var useCaseMatrix = map[string]map[string]map[string]bulkQueryGen.QueryGenerator
 			"influx-http":      influxdb.NewInfluxQLBareAggregateMax,
 		},
 	},
-	common.UseCaseGroupWindowTranspose: {
-		fmt.Sprintf("%s-%s", influxdb.Count, influxdb.LowCardinality): {
```

Comment:
Are these `LowCardinality` and `HighCardinality` constants needed anymore?

Reply:
Yep they are still used in the actual query generator files, https://github.com/influxdata/influxdb-comparisons/blob/master/bulk_query_gen/influxdb/influx_group_window_transpose_count.go for example